_posts/2017-08-10-rubyonrails.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Learn Rails and build web apps"
+date: 2017-08-10 00:00:00 +530
+author: Anant Prakash
+category:
+tags:
+- Ruby on Rails
+- Web development
+---
+
+## Hi reader !
+
+Most of you would have learnt/seen HTML & CSS based static web pages.<br>
```

Comment:
I think it makes sense to avoid `<br>` and use a newline instead, since we use markdowny of syntax everywhere else anyway. For example, 

```
a <br>
b
```

can be written as 

```
a

b
```

So this applies to all places I think.